containers/jetty/pom.xml

```diff
@@ -134,6 +134,20 @@
       <groupId>com.gu</groupId>
       <artifactId>kinesis-logback-appender</artifactId>
       <version>1.4.2</version>
+      <exclusions>
+        <exclusion>
+          <groupId>com.amazonaws</groupId>
+          <artifactId>aws-java-sdk-core</artifactId>
+        </exclusion>
+        <exclusion>
+          <groupId>com.amazonaws</groupId>
+          <artifactId>aws-java-sdk-kinesis</artifactId>
+        </exclusion>
+      </exclusions>
+    </dependency>
+    <dependency>
+      <groupId>com.amazonaws</groupId>
+      <artifactId>aws-java-sdk-kinesis</artifactId>
```

Comment:
since you're excluding both sdk-core and sdk-kinesis, let's add both as dependency instead of just kinesis (which I'm assuming has dependency on core but let's specify it explicitely).